src/Enum/CTypeEnum.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Matecat\SubFiltering\Enum;
+
+class CTypeEnum
+{
+    const ORIGINAL_PH = 'x-original_ph';
+    const HTML = 'x-html';
+    const TWIG = 'x-twig';
+    const RUBY_ON_RAILS = 'x-ruby-on-rails';
+    const SNAILS = 'x-snails';
+    const CURLY_BRACKETS = 'x-curly-brackets';
+    const PERCENT_SNAILS = 'x-percent-snails';
+    const PERCENT_NUMBER_SNAILS = 'x-percent-number-snails';
+    const PERCENTAGES = 'x-percentages';
+    const SPRINTF = 'x-sprintf';
+    const AIRBNB_VARIABLE = 'x-airbnb-variable';
```

Comment:
Change this name to a different one please

Reply:
Done.